### DISCUSSION TITLE
Book Announcement

### DISCUSSION THREAD

**Discussion by Alkibiades410** [Stellarium/stellarium#1815] [Show and tell]
**Just published**: the textbook definition of archeoastronomy! This book simply would **not** exist without the effort of everyone involved and many of Stellarium's capabilities. I devote five paragraphs to the program p. 280, outlining the features Greco-Roman historians will need and thanking everyone involved! I do so again here.

![ayp-cover](https://user-images.githubusercontent.com/35356700/126709821-4f3c5c0e-00a1-489a-ba2d-26e7228860b3.png)

Available from [Westphalia Press](https://westphaliapress.org/2021/06/25/the-athenian-year-primer/)

**Comment by Keatah** [comment]
Love it!
